app/uk/gov/hmrc/taxenrolmentassignmentfrontend/views/PTEnrolmentOnAnotherAccount.scala.html

```diff
@@ -16,47 +16,79 @@
+@(ptEnrolmentOtherAccountViewModel: PTEnrolmentOtherAccountViewModel)(implicit request: Request[AnyRef], messages: Messages, appConfig: AppConfig)
+
+@content = {
+    <a class = "govuk-link" href=@routes.ReportSuspiciousIDController.viewNoSA>@messages("ptEnrolmentOnOtherAccount.linkText")</a>
+}
 
 @layout(
     pageTitle = Some(messages("ptEnrolmentOnOtherAccount.title")), None) {
 
-    <h2 class="govuk-heading-xl">@messages("ptEnrolmentOnOtherAccount.heading")</h2>
+    <h1 class="govuk-heading-xl">@messages("ptEnrolmentOnOtherAccount.heading")</h1>
+
+    <p class="govuk-body">@messages("ptEnrolmentOnOtherAccount.paragraph1", ptEnrolmentOtherAccountViewModel.ptAccountDetails.userId)</p>
+    <p class="govuk-body">@messages("ptEnrolmentOnOtherAccount.paragraph2", ptEnrolmentOtherAccountViewModel.currentAccountDetails.userId)</p>
+
     <p class="govuk-body">@messages("ptEnrolmentOnOtherAccount.text1")
-        <a href="@routes.SignOutController.signOut" class="govuk-link--no-visited-state">@messages("ptEnrolmentOnOtherAccount.signinAgain")</a>
+        <a href="@routes.SignOutController.signOut" class="govuk-link--no-visited-state">@messages("ptEnrolmentOnOtherAccount.signInAgain")</a>
       @messages("ptEnrolmentOnOtherAccount.text2")
     </p>
+
     <dl class="govuk-summary-list">
-        @summaryRow("common.userId", ptAccountDetails.userId)
-        @if(ptAccountDetails.email.isDefined) {
-          @summaryRow("ptEnrolmentOnOtherAccount.email", ptAccountDetails.email.get)
+        @summaryRow("common.userId", ptEnrolmentOtherAccountViewModel.ptAccountDetails.userId)
+        @if(ptEnrolmentOtherAccountViewModel.ptAccountDetails.email.isDefined) {
+          @summaryRow("ptEnrolmentOnOtherAccount.email", ptEnrolmentOtherAccountViewModel.ptAccountDetails.email.get)
         }
-        @summaryRow("ptEnrolmentOnOtherAccount.lastLoginDate", ptAccountDetails.lastLoginDate)
-        @for(mfaDetail <- ptAccountDetails.mfaDetails) {
+        @summaryRow("ptEnrolmentOnOtherAccount.lastLoginDate", ptEnrolmentOtherAccountViewModel.ptAccountDetails.lastLoginDate)
+        @for(mfaDetail <- ptEnrolmentOtherAccountViewModel.ptAccountDetails.mfaDetails) {
         @summaryRow(mfaDetail.factorNameKey, mfaDetail.factorValue)
         }
     </dl>
 
-<!--//TODO remove fraud reporting url after consultation with leads, as it requires other changes such remove couple of tests etc etc-->
-    <p class="govuk-body">
-        <a href="@routes.ReportSuspiciousIDController.viewNoSA" class="govuk-link--no-visited-state">@messages("ptEnrolmentOnOtherAccount.notMyUserId")</a>
+    <div id="inset_text">
+
+        @govukInsetText(InsetText(
+        content = HtmlContent(content)
+        ))
+
+    </div>
+
+    @if(ptEnrolmentOtherAccountViewModel.saUserCred.isDefined){
+
+    <h3 class="govuk-heading-m">@messages("ptEnrolmentOnOtherAccount.saHeading")</h3>
```

Comment:
This needs to be a h2 instead of h3 since you changed the above header from h2 to h1. It will be flagged as an accessibility issue since it's not in the right order - https://dequeuniversity.com/rules/axe/4.3/heading-order?application=webdriverjs

Alternatively, it might be better to offload this to tech debt because it looks like the other pages are using a h2 for the main header and I assume we want to be consistent across the pages.